City-of-Helsinki/helfi-etusivu-news-search
# Uhf 5916 search results

# [UHF-5916](https://helsinkisolutionoffice.atlassian.net/browse/UHF-5916)
Updated the app to show list of results.

## What was done
- Developed the app to show results from the index
- Worked on the test suite
- Figured out how to use Drupal translations

## How to install
* Checkout branch. Run `nvm use; npm i; npm start`
* Check install instructions [here](https://github.com/City-of-Helsinki/drupal-helfi-etusivu/pull/91) for the Etusivu instance. You need to have those up and running so the index can be queried.

## How to test
* Run `npm t`. All tests should pass.
* Run `npm start`
* Check that app lists results from the index in the HDBT-defined style.


## Designers review
